release/ray_release/test.py

```diff
@@ -82,6 +116,57 @@ def get_anyscale_byod_image(self) -> str:
         """
         return f"{DATAPLANE_ECR}/{self.get_byod_repo()}:{self.get_byod_image_tag()}"
 
+    def update_from_s3(self) -> None:
+        """
+        Update test object with data field from s3
+        """
+        try:
+            data = (
+                boto3.client("s3")
+                .get_object(
+                    Bucket=AWS_BUCKET,
+                    Key=f"{AWS_KEY}/{self.get_name()}.json",
```

Comment:
I thought we agreed that for each test it is creating a new object in s3 only? why do we need updating the json here?

Reply:
the test object in s3 will also store other information that the test object at runtime doesn't have, such as the current state (jailed|failing|passing), last N results (the next PR computes this), current github issue, bisect job, etc. 

I can describe more in person maybe tomorrow